test/integration/controller/issuer/issuer_test.go

```diff
@@ -28,6 +28,8 @@ var _ = Describe("Issuer controller tests", func() {
 	)
 
 	BeforeEach(func() {
+		Expect(acmeDirectoryAddress).NotTo(BeEmpty())
```

Comment:
During development, I messed up initializing the package level variable in the test suite. Tests should assert early on that the variable has been set.